### DISCUSSION TITLE
Inquiry about DH test cases

### DISCUSSION THREAD

**Discussion by testter2** [openthread/openthread#9854] [Q&A]
Hello,

I'm currently trying to run DH test cases from latest GRL V59 (ex: DH 3.3 on SED role) but  I've encountered the following error: "'OpenThread' object has no attribute 'get_ed_omr_address'".

I can see latest openthread.py doesn't have such APIs get_ed_omr_address or srpCallBackEnable, so is this normal ?

The DUT is configured as Full Thread Device with openthread cli app so Do I miss any special procedure to run these tests or what's the required feature that should be enabled for these tests ?

Thanks

**Comment by jwhui** [comment]
This issue seems related to the Thread Group Test Harness. I suggest submitting a JIRA under the Thread Test Bed (TSTBED) project.
